lib/Config.py

```diff
@@ -82,3 +84,19 @@ def getFile(cls, getfile, unpack=True):
           if len(fzip.namelist())>0:
             data=BytesIO(fzip.read(fzip.namelist()[0]))
     return (data, response)
+  
+  # Get Microsoft Bulletin
```

Comment:
@IrootGeek sure, but it feels weird tome to  have a generic Config script with some generic fetch code that also has some very specific msrc API code.